scripts/utilities/package_dataset_for_runpod.py

```
@@ -0,0 +1,754 @@
+
+
+def create_meta_jsonl(base_path, splits_with_captions):
+    """Create meta.jsonl file with dataset metadata"""
+    meta_path = base_path / "meta.jsonl"
+
+    # Track processed filenames to match the deduplication in copy function
+    processed_filenames = set()
+
+    with open(meta_path, "w", encoding="utf-8") as f:
+        for split_name, image_caption_pairs in splits_with_captions.items():
+            for image_path, caption in image_caption_pairs:
```

Ruff (0.12.2)

Loop control variable `caption` not used within loop body

Rename unused `caption` to `_caption`

(B007)